src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -378,12 +375,23 @@ describe(`uploadModal `, () => {
     });
 
     it("can clear timer when download action is complete", async () => {
-        const store = createStore();
+        const store = createStore(fakeMetadata, {
+            summary: {complete: false, downloading: false } as any,
+            spectrum: {complete: false, downloading: false} as any
+        });
```

Comment:
Using local metadata here, as we will be editing to set complete in spectrum to true later in the test. Failing to do this was what caused the errors I had thought were down to a race condition!